### ISSUE TITLE
sidebar 社交那块排列有问题

### ISSUE THREAD

**Issue by bihell** [iissnan/hexo-theme-next#1806]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.iissnan.com/).
- [x] And yes, I already searched for current [issues](https://github.com/iissnan/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 

用了最新发布的版本.正常社交那块应该是两两排列的
https://almostover.ru

### Actual behavior 

但我这边..特别诡异
http://www.bihell.com

### Steps to reproduce the behavior 
Link to demo site with this issue: N/A
Link(s) to source code or any usefull link(s): N/A

### NexT Information

**NexT Version:**

```
[ ] Master
[x] Latest Release
[ ] Old version - 
```

**NexT Scheme:**

```
[x] All schemes
[ ] Muse
[ ] Mist
```

### Other Information

**Comment by bihell** [comment]
@aixiu 我知道问题在哪里了...浏览器的问题.我用Safari就会碰到这个问题.看来部分浏览器不兼容..
<img width="248" alt="2017-08-08 11 41 11" src="https://user-images.githubusercontent.com/19926035/29080778-48c8c2d8-7c93-11e7-920b-de6bc4541fab.png">

**Comment by aixiu** [comment]
@bihell 我用的火狐 ，谷歌，都没。那你可以用英文提问，让他修复！！中文他看不明白。

**Comment by bihell** [comment]
@aixiu  = =....作者不是国人啊..等下有时间改下...

**Comment by ivan-nginx** [comment]
@bihell what's the problem? Try to use `icons_only` option. Maybe flex or any CSS3 styles not work on old browsers. Anyway, Safary is not best browser. But also, if anything work on my AO site - this may be fixed in current master branch.

**Comment by bihell** [comment]
@ivan-nginx  It's a arrangement  problems  of social area.  When user use **Safari**, the problem will
 appear .    I'v checked with other guys,Same problem here.
<img width="248" alt="2017-08-08 11 41 11" src="https://user-images.githubusercontent.com/19926035/29152853-402fc50c-7dbc-11e7-9524-329e20c52c99.png">

**Comment by ivan-nginx** [comment]
@bihell only on safary? And only on your site or clean test latest master branch demo site too?

**Comment by ivan-nginx** [comment]
Fixed in 83c2b84 for **[Safari 9.x and later](https://www.browserstack.com/screenshots/d4d932d1803ab0c7b36a86bcbd455d3e942ecb33)**.

@bihell if solved, close it.

**Comment by bihell** [comment]
OK, Thans a lot.
